### DISCUSSION TITLE
[Bug]: Permissions in America

### DISCUSSION THREAD

**Discussion by ahnetzley** [neurodesk/neurodesk.github.io-hugo#444] [Bug Report]
### What platform are you seeing the problem on?

HPC

### What happened

I use Neurodesk in the browser as I cannot install on my Silicon Mac. This week I have been unable to upload data to Neurodesk if I select America. I can upload if I select Europe instead, but overall operation is much slower. It says I do not have permission to upload. Does anyone know how I may be able to resolve this? I should be using America if possible as I am located in the US.

**Comment by stebo85** [comment]
This seems to be a bug - we are investigating

**Comment by stebo85** [comment]
Dear @ahnetzley - we checked everything on our side and there is nothing wrong and the upload is working as expected.

I had a quick look in your account and your homedirectory is full! You can only store 2GB of files on the play service. You need to make sure that when you are deleting files that you are also removing them from the Trash.

```
rm -rf ~/.local/share/Trash/files/
```

Then the upload is also working again for you.

**Comment by stebo85** [comment]
Dear @ahnetzley - we saw that that you tried to download the FSL course material, but you hit the 2GB homedirectory limit again. Would it useful for you to have access to these course files inside Neurodesk play? If yes we could look into making these available for you :)

**Reply by ahnetzley** [reply]
Hi Steffen, it would be very useful to have the FSL course material available. I am trying DTI analysis this week, so I was able to isolate the subj1_preproc directory used in the practical and upload that to Neurodesk play. I should be set for now, but in general I was thinking it would be nice to have

**Reply by stebo85** [reply]
ok, good to know! We get to work on this and see what we can do. We will also look into increasing the home directory limit to 10GB and get more resources :)

**Comment by stebo85** [comment]
Dear @ahnetzley - instead of providing course material to everyone we made all homedirectories larger - let us know if you still run into any issues.
